internal/gengapic/genrest.go

```diff
@@ -629,24 +627,30 @@ func (g *generator) emptyUnaryRESTCall(servName string, m *descriptor.MethodDesc
 	// TODO(dovs): handle http headers
 	// TODO(dovs): handle deadlines
 	// TODO(dovs): handle call options
-	p("// The default (false) for the other options are fine.")
-	p("// Field names should be lowerCamel, not snake.")
-	p("m := protojson.MarshalOptions{AllowPartial: true, EmitUnpopulated: true, UseProtoNames: false}")
-	requestObject := "req"
-	if info.body != "" && info.body != "*" {
-		requestObject = "body"
-		p("body := req.Get%s()", snakeToCamel(info.body))
+	body := "nil"
```

Comment:
nit: add comment "Marshal body for HTTP methods that take a body" Below as well.

Reply:
Done.